help/en/package/apps/TabbedPreferences.shtml

```diff
@@ -417,13 +417,30 @@
       <h2 id="Messages">Messages</h2>
 
       <div>
-        <a href="images/PrefsMessages.png"><img src="images/PrefsMessages.png" height="194" width=
-        "479" class="floatRight" alt="JMRI v4.17 Messages Prefs tab"></a>
-        <p>This allows you to change the way that information and warning messages are displayed in
-        JMRI.</p>
+        <p>Some of the message dialogs include an option to ignore the message in the future. They
+        provide a checkbox with either <em>Skip message in future?</em> or <em>Remember this setting
+        for next time?</em> as the description.  If the checkbox is selected, the dialog will no
+        longer appear. A second dialog is displayed with a reminder to use this preference to
+        re-display the suppressed message.  This reminder can also be suppressed.</p>
+
+        <p>The Messages preference provides the ability to <strong>undo</strong> the selections.  It
+        can also be used to manually set the preferred settings.</p>
+
+        <p>The Messages preference will show 17 tabs, one for each standard item such sensor or
```

Comment:
A suggestion is to change `17` to `many`. That way, we can add new tabs if desired without needing to update this number.

Reply:
I used the tab count to quantify the large number of standard tags.  It is better than just saying "many" or "a lot".

The number of tabs is controlled by UserMessagePreferencesPane so a change can be reflected in the help page.  

By the way, now that the process is actually documented, the central location for LogixNG is not necessary.  There is no reason to change since it works fine as is.